apps/native-component-list/App.tsx

```diff
@@ -2,27 +2,36 @@ import { AppLoading } from 'expo';
 import * as React from 'react';
 import { Platform, StatusBar } from 'react-native';
 import { AppearanceProvider } from 'react-native-appearance';
-import { SafeAreaProvider } from 'react-native-safe-area-context';
 import { enableScreens } from 'react-native-screens';
 
 import RootNavigation from './src/navigation/RootNavigation';
 import loadAssetsAsync from './src/utilities/loadAssetsAsync';
 
-if (Platform.OS === 'android') {
-  enableScreens(true);
+if (Platform.OS === 'web') {
+  const originalWarn = console.warn;
+  console.warn = (...props: any[]) => {
+    // Mute the warnings about react-native-web 11 for now.
+    // This warning is thrown whenever ScrollView or TouchableWithoutFeedback are used.
+    // Remove this after support for react-native-web@0.12 has landed.
+    for (const message of props) {
+      if (
+        typeof message === 'string' &&
+        message.includes(
+          'has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles'
+        )
+      ) {
+        return;
+      }
+    }
+    originalWarn(...props);
+  };
 }
 
-export default function AppContainer() {
-  return (
-    <SafeAreaProvider>
-      <AppearanceProvider>
-        <App />
-      </AppearanceProvider>
-    </SafeAreaProvider>
-  );
+if (Platform.OS === 'android') {
+  enableScreens(true);
```

Comment:
Currently React Navigation only uses screens on Android so the conditional is nut needed if you're not using screens elsewhere.